### DISCUSSION TITLE
Schema Registry Config connectivity issue with SSL

### DISCUSSION THREAD

**Discussion by chethan-bs** [open-metadata/OpenMetadata#7633] [Q&A] [answered]
We are trying to connect Schema Registry via SSL and unable to connect with SSL.

Unable to pass link below :

         --property schema.registry.url=https://<Schema Registry  URL> \
         --property schema.registry.ssl.keystore.location=sr.keystore.jks \
         --property schema.registry.ssl.truststore.location=sr.truststore.jks \
         --property schema.registry.ssl.keystore.password=*** \
         --property schema.registry.ssl.truststore.password=*** 

could you please let us know how what is parameter need to be passed in properties in schemaRegistryConfig

<img width="453" alt="image" src="https://user-images.githubusercontent.com/110088247/191517023-f516b316-ba64-451f-ab39-ed7714826354.png">

**Comment by pmbrull** [comment, marked as answer]
Hi @chethan-bs, sorry that we missed your message. It might be easier of you reached out to us in Slack https://slack.open-metadata.org/ and shared some more details on your deployment and logs. Thanks
